rust/lance-index/src/scalar/inverted/tokenizer.rs

```diff
@@ -203,6 +211,11 @@ impl InvertedIndexParams {
         self
     }
 
+    pub fn custom_stop_words_file(mut self, custom_stop_words_file: Option<String>) -> Self {
+        self.custom_stop_words_file = custom_stop_words_file;
+        self
+    }
```

Comment:
You should add a docstring here, since this is the public method. Could you make sure to describe the format the file should be in?